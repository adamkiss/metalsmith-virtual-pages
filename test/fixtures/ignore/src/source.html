---
title: File removed from tree after generation
---

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>This file should not exist</title>
</head>
<body>
  o/
</body>
</html>