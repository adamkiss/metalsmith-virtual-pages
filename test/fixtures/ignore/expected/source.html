<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Source should be kept</title>
</head>
<body>
  o/
</body>
</html>